docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
+git stash pop
+docker-compose build
+docker-compose up -d
+docker-compose run --rm web rails assets:precompile
+docker-compose run --rm web rails db:migrate
```

Comment:
Thank you for commenting. the db:migrate task is taken like this straight drom the [docker update guide](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Docker-Guide.md) so I am assuming it safe. I have run this script for the past month via cron and did not have a problem with it.